OneCommunityGlobal/HGNRest
# Vishala Add Material

# Description
Add Materials Page
Or Implements #  (WBS) 3.3.7 (Phase 2)

## Related PRS (if any):
To test this backend PR you need to checkout the [#1748](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1748) frontend PR.

## Main changes explained:
- Worked on 2 collections for Adding Materials - Materials and Units

## How to test:
1. check into current branch
2. do `npm run build` and `npm run start` to run this PR locally
3. Refer front end PR for instructions
